pages/add-clinic.vue

```diff
@@ -141,8 +136,24 @@ export default {
         alert("Please fill out all of the fields!");
       }
     },
-    validate() {
-      this.$refs.form.validate();
+    errorMessages(field, name) {
+      if (field.$dirty && field.$invalid) {
+        return field
```

Comment:
Can we break this up to be more readable?
something like:

```
    const whatYourFilteringHere = field.$flattenParams().filter((param) => !field[param.name])
    const whatTheMapFunctionIsReturning = whatYourFilteringHere
    
    return whatTheMapFunctionIsReturning 
```

It's a good practice to put your return on a separate line if it's not clear what is being returned. 
(something like `return 25 + x` is easy to read)

Reply:
Split up the filter and map lines.